src/components/AvatarWrapper/AvatarWrapper.vue

```diff
@@ -136,8 +139,12 @@ export default {
 
 	setup() {
 		const isDarkTheme = useIsDarkTheme()
+
+		const failed = ref(false)
```

Comment:
Probably we should also reset it on token/source update. Otherwise the component doesn't support re-rendering with new props after it failed loading.

Reply:
Or the component must be used with `key`

Reply:
watch after avatarUrl would be enough, I think

Reply:
Yes, if you mean reset failed in watch for avatarUrl